packages/paste-style-props/src/types/typography.ts

```diff
@@ -55,4 +57,5 @@ export interface TypographyProps {
   textTransform?: TextTransform;
   whiteSpace?: WhiteSpace;
   wordBreak?: WordBreak;
+  wordWrap?: WordWrap;
```

Comment:
Is there a prop-type that also needs this? I can't remember if we do prop types for typography

Reply:
Yep, but forgot to add it to the typography prop types, so good catch. I just added it.